src/templates/next/components/shared/Notification/index.ts

```diff
@@ -0,0 +1 @@
+export { default } from "./Notification"
```

Comment:
question: this component isn't being used in any of the layouts, should it be added into the Skeleton layout?

Reply:
[88000ab](https://github.com/isomerpages/isomer-components/pull/67/commits/88000ab71881e74a26f77b82c12b38498b6edca1)